sdk/monitor/azquery/custom_client.go

```diff
@@ -74,3 +74,56 @@ func (e *ErrorInfo) UnmarshalJSON(data []byte) error {
 func (e *ErrorInfo) Error() string {
 	return string(e.data)
 }
+
+// Row of data in a table, types of data used by service specified in LogsColumnType
+type Row []any
+
+// Table - Contains the columns and rows for one table in a query response.
+type Table struct {
+	// REQUIRED; The list of columns in this table.
+	Columns []*Column `json:"columns,omitempty"`
+
+	// REQUIRED; The name of the table.
+	Name *string `json:"name,omitempty"`
+
+	// REQUIRED; The resulting rows from this query.
+	Rows []Row `json:"rows,omitempty"`
+
+	// internal map used to lookup column index
+	columnIndexLookup map[string]int
```

Comment:
We try to avoid mixing internal/external state in POD types as it breaks the ability to create fake instances of them.  And given the ambiguity problem mentioned below, it might just be easier to export this field.

Reply:
fixed in commit [2613738](https://github.com/Azure/azure-sdk-for-go/pull/19374/commits/2613738f8d37eec9188058fac45247e1ee89a4cc)